templates/manager/reports/url_report_data.html

```diff
@@ -0,0 +1,67 @@
+{% load filters %}
+<div class="card-header mt-4 bg-secondary mb-2 border-0">
+  <div class="row">
+    <div class="col-9">
+     <h3 class="h6 mb-0">URL</h3>
+    </div>
+    <div class="col-2">
+      <h3 class="h6 text-center mb-0">Clicks</h3>
+    </div>
+  </div>
+</div>
+<div id="accordion" role="tablist" class="mb-3">
+  {% for url in data %}
+    <div class="report-data-card card mb-1">
+      <div class="card-header bg-secondary" role="tab" id="heading{{ url.id }}">
+        <a class="collapsed text-secondary" data-toggle="collapse" href="#collapse{{ url.id }}" aria-expanded="false" aria-controls="collapse{{ url.id }}">
+          <div class="row">
+            <div class="report-data-url col-9">
+              {{ url.name }}<a class="text-default ml-2" href="{{ url.name }}" target="_blank"><span class="fas fa-external-link-alt" aria-hidden="true"></span></a>
+            </div>
+            <div class="col-2 text-center">{{ url.total_clicks }}</div>
+            <div class="col-1">
+              <a class="collapsed text-secondary" data-toggle="collapse" href="#collapse{{ url.id }}" aria-expanded="false" aria-controls="collapse{{ url.id }}"><span class="fas fa-chevron-down" aria-hidden="true"></span></a>
+            </div>
+          </div>
+        </a>
+      </div>
+      <div id="collapse{{ url.id }}" class="collapse" role="tabpanel" aria-labelledby="heading{{ url.id }}" data-parent="#accordion">
+        <div class="pt-0">
+          <div class="bg-faded card-block">
```

Comment:
Some phrasing that helps connect the expanded URL to the information in the collapse box could be helpful to include at the top of this box--maybe something like "URL was included in the following email instance:"